spec/index.bs

```diff
@@ -2282,6 +2282,46 @@ dictionary GPUImageBitmapCopyView {
 
   * {{GPUImageBitmapCopyView/origin}}: If unspecified, defaults to `[0, 0]`.
 
+### <dfn method for=GPUCommandEncoder>copyBufferToBuffer(source, sourceOffset, destination, destinationOffset, size)</dfn> ### {#GPUCommandEncoder-copyBufferToBuffer}
+
+<div algorithm="GPUCommandEncoder.copyBufferToBuffer">
+
+  **Arguments:**
+    - {{GPUBuffer}} |source|
+    - {{GPUSize64}} |sourceOffset|
+    - {{GPUBuffer}} |destination|
+    - {{GPUSize64}} |destinationOffset|
+    - {{GPUSize64}} |size|
+
+  **Returns:** void
+
+  Encode a command into the {{GPUCommandEncoder}} that copies |size| bytes of data from the |sourceOffset| of a {{GPUBuffer}} |source| to the |destinationOffset| of another {{GPUBuffer}} |destination|.
+
+</div>
+
+<div algorithm class=validusage>
+
+  <dfn abstract-op>copyBufferToBuffer Valid Usage</dfn>
+
+  Given a {{GPUCommandEncoder}} |encoder| and the arguments {{GPUBuffer}} |source|, {{GPUSize64}} |sourceOffset|, {{GPUBuffer}} |destination|, {{GPUSize64}} |destinationOffset|, {{GPUSize64}} |size|, the following validation rules apply:
+
+  - |encoder|.{{GPUCommandEncoder/copyBufferToBuffer()}} must be called before calling |encoder|.{{GPUCommandEncoder/finish()}}. // TODO: define the state machine for GPUCommandEncoder.
```

Comment:
Do we need this specifically, as opposed to "`GPUCommandEncoder` should be valid"? When `finish()` is called, we should be able to say that the command encoder becomes invalid, since it's technically consumed at that point (in a different PR, not necessary here).

Reply:
Done